js/flightlog_fields_presenter.js

```diff
@@ -431,16 +431,35 @@ function FlightLogFieldPresenter() {
 
         DEBUG_FRIENDLY_FIELD_NAMES = {...DEBUG_FRIENDLY_FIELD_NAMES_INITIAL};
 
-        if (firmwareType === FIRMWARE_TYPE_BETAFLIGHT && semver.gte(firmwareVersion, '4.3.0')) {
-            DEBUG_FRIENDLY_FIELD_NAMES.FF_INTERPOLATED = {
-                'debug[0]':'Raw FF Derivative [Roll]',
-                'debug[1]':'Cleaned FF Derivative ',
-                'debug[2]':'Cleaned Boost Amount [Roll]',
-                'debug[3]':'Duplicate Marker',
-            };
+        if (firmwareType === FIRMWARE_TYPE_BETAFLIGHT) {
+            if (semver.gte(firmwareVersion, '4.2.0')) {
```

Comment:
This is problematic this way, as you are executing this part for 4.3 as well, unintentionally adding `DEBUG_FRIENDLY_FIELD_NAMES.FF_INTERPOLATED`.
A better way to do this will be to go from more restrictive condition to less, with `else`:

```
if (semver.gte(firmwareVersion, '4.3.0')) {
[stuff for >= 4.3 only]
} else if (semver.gte(firmwareVersion, '4.2.0')) {
[stuff for 4.2 only]
} else {
[stuff for < 4.2]
}
```

Reply:
I've hopefully addressed this now.